Gemfile

```diff
@@ -30,7 +30,7 @@ gem "oa-openid", :require => "omniauth/openid"
 gem 'activerecord-session_store'
 gem 'actionpack-action_caching'
 gem 'rails-observers'
-
+gem "cancancan"
```

Comment:
I'd rather specify versions of gems when I use them. However, this is not applied in the rest of the file so I think it's OK.